changelog/17.0/17.0.0/summary.md

```diff
@@ -257,6 +259,14 @@ These stats are deprecated in v17.
 | `backup_duration_seconds` | `BackupDurationNanoseconds` |
 | `restore_duration_seconds` | `RestoreDurationNanoseconds` |
 
+### <a id="vtctld"/> Vtctld
+
+#### <a id="vtctld-deprecated-flags"/> Deprecated Flags
+
+The flag `schema_change_check_interval` used to accept either a Go duration value (e.g. `1m` or `30s`) or a bare integer, which was treated as seconds.
+This behavior was deprecated in v15.0.0 and has been removed.
```

Comment:
I hope we announced the deprecation in v15

Reply:
the deprecation shipped in 15.0 and has been dutifully issuing depreciation warnings ever since